src/contracts/middleware/BLSIndexRegistryCoordinator.sol

```diff
@@ -1,17 +1,18 @@
 // SPDX-License-Identifier: BUSL-1.1
 pragma solidity =0.8.12;
 
-import "../interfaces/IRegistryCoordinator.sol";
-import "../interfaces/IBLSPubkeyRegistry.sol";
+import "../interfaces/IBLSStakeRegistryCoordinator.sol";
 import "../interfaces/IIndexRegistry.sol";
 
 import "../libraries/BytesArrayBitmaps.sol";
 
 import "./StakeRegistry.sol";
 
-contract BLSIndexRegistryCoordinator is StakeRegistry, IRegistryCoordinator {
+contract BLSIndexRegistryCoordinator is StakeRegistry, IBLSStakeRegistryCoordinator {
```

Comment:
This naming scheme doesn't make sense to me.  BLSRegistryCoordinator seems to be more appropriate.  Same goes for the interface....